src/test/java/weblogic/logging/exporter/config/MapUtilsTest.java

```diff
@@ -41,6 +44,60 @@ public void whenStringArrayValueIsList_returnAsArray() {
     assertThat(MapUtils.getStringArray(map, "values"), arrayContaining("7", "8", "true"));
   }
 
+  @DisplayName("Check getBooleanValue for true values")
+  @Test
+  public void checkGetBooleanForTrueValues() {
+    Map<String, Object> map = new HashMap<>();
+    map.put("1", "true");
+    map.put("2", "t");
+    map.put("3", "yes");
+    map.put("4", "on");
+    map.put("5", "y");
+    map.put("6", "truenot");
+    map.put("7", "nottrue");
+    map.put("8", "yesss");
+    map.put("9", "y ");
+    map.put("10", " y ");
+
+    assertEquals(true, MapUtils.getBooleanValue(map, "1"));
+    assertEquals(true, MapUtils.getBooleanValue(map, "2"));
+    assertEquals(true, MapUtils.getBooleanValue(map, "3"));
+    assertEquals(true, MapUtils.getBooleanValue(map, "4"));
+    assertEquals(true, MapUtils.getBooleanValue(map, "5"));
+    assertThrows(ConfigurationException.class, () -> MapUtils.getBooleanValue(map, "6"));
+    assertThrows(ConfigurationException.class, () -> MapUtils.getBooleanValue(map, "7"));
+    assertThrows(ConfigurationException.class, () -> MapUtils.getBooleanValue(map, "8"));
+    assertThrows(ConfigurationException.class, () -> MapUtils.getBooleanValue(map, "9"));
+    assertThrows(ConfigurationException.class, () -> MapUtils.getBooleanValue(map, "10"));
+  }
+
+  @DisplayName("Check getBooleanValue for false values")
+  @Test
```

Comment:
Again, I'd split this into two tests, picking good names for each. You might also consider making the map a field in the test, as it is used so often.